xml/System.Composition.Convention/PartConventionBuilder`1.xml

```diff
@@ -66,6 +66,7 @@
         <summary>Selects a property on the part to export.</summary>
         <returns>A part builder allowing further configuration of the part.</returns>
         <remarks></remarks>
+        <exception cref="T:System.Argument"><paramref name="propertySelector" /> The expression must be <see cref="T:System.Linq.Expressions.MemberExpression"/> for accessing a property.</exception>
```

Comment:
`<paramref>` is a general formatting tag for use in text. So the exception description should read, "The `<paramref name="propertySelector" />` expression must be a `<see cref="T:System.Linq.Expressions.MemberExpression"/>` for accessing a property." That's true of each of the uses of the paramref element here.

Also, in each case, "a" should precede the type name ("a `<see cref="T:System.Linq.Expressions.MemberExpression"/>`"). There are a few cases where you've done that, but it should be done in all cases.